app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class User < ApplicationRecord
+  has_many :post
```

Comment:
By convention, this should've been named `posts` and not `post`. We should be able to execute `user.posts` and obtain all posts made by an user. At the moment, we'd have to use `user.post` and that doesn't sound right because we expect a collection of posts (plural) and not just a post (singular). Using the singular version of posts as post is misleading because it gives the impression we are talking about a One-to-One relationship, when in reality this is a One-to-Many relationship:

```
irb(main):004:0> u.posts
Traceback (most recent call last):
        2: from (irb):4
        1: from (irb):4:in `rescue in irb_binding'
NoMethodError (undefined method `posts' for #<User:0x000055a45e3c3c48>)
irb(main):005:0> u.post
  Post Load (39.2ms)  SELECT  "posts".* FROM "posts" WHERE "posts"."user_id" = $1 ORDER BY "posts"."created_at" DESC LIMIT $2  [["user_id", 1], ["LIMIT", 11]]
=> #<ActiveRecord::Associations::CollectionProxy [#<Post id: 1, description: "Hello World!", user_id: 1, created_at: "2019-10-01 00:27:53", updated_at: "2019-10-01 00:27:53">]>
```

If you look closely, the SQL query indicates `select "posts"`.

I'm marking this as optional as its from a previous milestone and does not affect current functionality of your app; however, I strongly encourage you to make the modification.

[`OPTIONAL`]